xml-conduit/Text/XML/Stream/Parse.hs

```diff
@@ -406,23 +409,39 @@ data ParseSettings = ParseSettings
     -- Default: False
     --
     -- Since 1.2.1
+    , psPreserveWhiteSpace :: Bool
+    -- ^ Whether we should preserve literal XML whitespace within nodes.
+    --
+    -- Default: False
```

Comment:
if we preserve whitespace by default, then using streaming `tag` combinators becomes more difficult, since they will have to consume whitespace, and it is impossible to know if it is XML literal whitespace (and thus reasonable to skip) or decoded XML entities (and thus illegal to skip).

This is not my favoured route, for this very reason, as mentioned on the issue, but it does limit the scope of changes.

Thanks for taking some time to review the changes.